include/EVT/utils/bits.hpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * Provides a series of utilities for operating on the binary representation
+ * of a number. These utilites are commonly used operations that may be needed.
+ */
+
+#ifndef _BITS_H_
+#define _BITS_H_
+
+#include <cstdint>
+
+// 16 bit operations
+/**
+ * Get the high byte from the number.
+ *
+ * @param data The number to get the high byte from
+ * @return The high order byte
+ */
+constexpr static inline uint8_t getHighByte(uint16_t data)
+{
+    return static_cast<uint8_t>((data & 0xFF00) >> 8);
```

Comment:
Shouldn't function definitions be in the source file instead of header file?  Or is this normal practice for constexpr?

Reply:
Since they are inline, they don't operate like normal functions. Instead of the call generating a jump statement to the function. The code is expanded inline where there is the call. That is why the code is in the header.